Product/Production/Services/DocumentSubmissionCore/src/test/java/gov/hhs/fha/nhinc/docsubmission/outbound/deferred/request/StandardOutboundDocSubmissionDeferredRequestTest.java

```diff
@@ -67,61 +70,99 @@
  */
 public class StandardOutboundDocSubmissionDeferredRequestTest {
 
-    protected Mockery context = new JUnit4Mockery() {
-        {
-            setImposteriser(ClassImposteriser.INSTANCE);
-        }
-    };
-    final XDRAuditLogger mockXDRLog = context.mock(XDRAuditLogger.class);
-    final XDRPolicyChecker mockPolicyCheck = context.mock(XDRPolicyChecker.class);
-    final SubjectHelper mockSubjectHelper = context.mock(SubjectHelper.class);
-    final OutboundDocSubmissionDeferredRequestDelegate mockDelegate = context.mock(OutboundDocSubmissionDeferredRequestDelegate.class);
+    private final String senderHCID = "1.1";
```

Comment:
A "private final" variable should always be static if it's declared and assigned in the same line.